tasks/process-spectrum.js

```diff
@@ -169,7 +169,8 @@ async function processComponent(componentPath) {
      */
     for await (const conversion of conversions) {
         const sourcePath = require.resolve(conversion.inPackage);
```

Comment:
Should this be `const sourcePath = require.resolve(conversion.inPackage).replace('index.css', 'index-base.css');`?

Reply:
Updated this... However after changing it I am seeing some un-expected UI changes. Tried to trace down why but haven't figured out properly yet.

Reply:
The system-overrides to be imported after each component's index-base css. Otherwise the UI will be off.

Reply:
I updated it thnx. Please verify.